src/shared/components/plots/ScatterPlot.tsx

```diff
@@ -163,18 +163,21 @@ export default class ScatterPlot<D extends IBaseScatterPlotData> extends React.C
     private get correlationInfo() {
         const approxTextWidth = 107; // experimentally determined
         const x = this.legendX;
+        const style = {fontFamily: "Arial, Helvetica", fontSize: "13px"};
```

Comment:
ah okay so thats the problem. `correlationInfo` is used in `render` outside the chart, it has to be because `VictoryChart` cant handle a `<g>` as a child.